apps/myaccount/src/hooks/use-sign-in.ts

```diff
@@ -127,6 +130,19 @@ const useSignIn = (): UseSignInInterface => {
             window[ "AppUtils" ].updateTenantQualifiedBaseName(tenantDomain);
         }
 
+        let orgType: OrganizationType;
```

Comment:
This whole logic is defined inside `useGetCurrentOrganization` hook. Can we use that here?

Reply:
useGetCurrentOrganization is used in console.